tools/modules/framework/manuals_help_lists.sh

```
@@ -0,0 +1,51 @@
+framework_options+=(
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)